src/plugins/controls/public/control_group/init_controls_manager.ts

```diff
@@ -166,18 +165,19 @@ export function initControlsManager(
           references.push(...controlReferences);
         }
 
-        explicitInputPanels[id] = {
+        panels.push({
+          id,
           grow,
           order: index,
           type: controlApi.type,
           width,
-          /** Re-add the `explicitInput` layer on serialize so control group saved object retains shape */
-          explicitInput: { id, ...rest },
-        };
+          /** Re-add the `embeddableConfig` layer on serialize so control group saved object retains shape */
```

Comment:
Why the rename of `explicitInput` to `embeddableConfig`. The controls are not embeddables. If we are renaming, how about just `control`?

Reply:
This was renamed to better align with the `embeddableConfig` key in the `panels` array. But yeah, as far as users are concerned, controls are not really embeddables since, unlike panels, controls are locked in place on the dashboard. Maybe it could be shortened to just `config` or `controlConfig`?

Reply:
Opened a discuss issue for a wider audience. https://github.com/elastic/kibana/issues/194757